SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/Mappings/v1/Willow/mapped_v1_dtdlv2_Willow.json

```diff
@@ -806,7 +806,7 @@
       "IsIgnored": true
     },
     {
-      "InputDtmi": "dtmi:org:brickschema:schema:Brick:Air_Quality_Sensing_Device;1",
+      "InputDtmi": "dtmi:mapped:core:Air_Quality_Sensing_Device;1",
```

Comment:
Should these changes be reflected in the REC mapping as well?

Reply:
I believe REC mappings should not reflect these changes because we are still referring to Brick namespaces in several instances.

Reply:
Sorry for being imprecise. I mean the files found [here](https://github.com/Azure/opendigitaltwins-tools/tree/41bcd128e470f565b645a84af79601c61ed1ab47/SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/Mappings/v0/BrickRec) and [here](https://github.com/Azure/opendigitaltwins-tools/tree/41bcd128e470f565b645a84af79601c61ed1ab47/SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/Mappings/v1/BrickRec) as they reference `dtmi:org:brickschema:schema:Brick:Air_Quality_Sensing_Device;1` and others similarly renamed.

Reply:
Thanks for the clarification. It appears that it is safe to change these mappings. Concepts such as `Air_Quality_Sensing_Device` are not represented by the Brick ontology and only exist in the Mapped ontology. There were several other instances like this, which have been corrected in this PR.

Reply:
Gotcha, thanks for identifying the gaps. Can you please make those changes in the other files in this PR as well?

Reply:
Yes, I can make these changes tomorrow

Reply:
Hi @Tyler-Angell 

Do you meant https://github.com/Azure/opendigitaltwins-tools/blob/bd9e0fb6d98643009fbe99a9b58bf49df1d24fc0/SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/Mappings/v1/BrickRec/mapped_v1_dtdlv2_Brick_1_3-REC_4_0.json by the other files?

Reply:
@Tyler-Angell I am finding the following files are using the incorrect DTMIs, would you like me to update all impacted files with the new remaps?:
- [mapped_json_v0_dtdlv2_Brick_1_3-REC_4_0.json](https://github.com/mapped/opendigitaltwins-tools/blob/fix-interface-remaps/SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/Mappings/v0/BrickRec/mapped_json_v0_dtdlv2_Brick_1_3-REC_4_0.json)
- [mapped_json_v0_dtdlv3_Brick_1_3-REC_4_0.json](https://github.com/mapped/opendigitaltwins-tools/blob/fix-interface-remaps/SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/Mappings/v0/BrickRec/mapped_json_v0_dtdlv3_Brick_1_3-REC_4_0.json)
- [mapped_json_v0_dtdlv2_Willow.json](https://github.com/mapped/opendigitaltwins-tools/blob/fix-interface-remaps/SmartPlaces.Facilities/lib/OntologyMapper.Mapped/src/Mappings/v0/Willow/mapped_json_v0_dtdlv2_Willow.json)

Reply:
@connorjcantrell - Yes please

Reply:
@jbkoh - I was asking to take a look at all the mapping files in the repo to as they were also either improperly configured initially or have been introduced/deprecated since (speaking generically here, not just for this PR). The desire is to keep all consumers of these files up to date, not just Willow 😊

Reply:
@Tyler-Angell Just finished updating the invalid mappings

Reply:
Looks like I'm getting duplicates, I'll have to look into it